projects/ion/src/lib/input-area/input-area.component.spec.ts

```diff
@@ -64,4 +65,19 @@ describe('InputAreaComponent', () => {
     const element = screen.getByTestId('input-area');
     expect(element).toBeDisabled();
   });
+
+  it('should event output', async () => {
```

Comment:
```suggestion
  it('should emit event with value typed', async () => {
```
You can give a more suggestive name to this case test.